ui/src/app/admin/controllers/AdminCtrl.spec.js

```diff
@@ -50,6 +50,13 @@ describe('Controller: Admin controller', function() {
 
     spyOn(AdminService, 'approve').and.callFake(fakeHttpPromise);
   
+
+    var user = {
+        user_name: 'BassaUser',
+        email: 'user@bassa.com',
+        password: 'bassapass'
+    }
+
     scope.approve();
```

Comment:
You need to pass the user object to the approve function
`scope.approve(user);`

Reply:
Done